src/script.js

```diff
@@ -1,21 +1,27 @@
 import { EventKey } from "./consts";
 import handleFormSubmit from "./handleFormSubmit";
 
-const attachEventListeners = () => {
-  const inputElement = document.getElementById("input");
-  const buttonElement = document.getElementById("button");
+let inputElement, buttonElement, amountFoundElement;
```

Comment:
Do they need to be globally defined?
Maybe init function should return them or smth